ghost/core/core/server/services/comments/CommentsController.js

```diff
@@ -71,7 +80,15 @@ module.exports = class CommentsController {
                 frame.options.filter = `post_id:${frame.options.post_id}`;
             }
         }
+
         frame.options.isAdmin = true;
+        // Admin routes lack member context due to cross-domain constraints (CORS), which prevents
+        // credentials from being passed. This causes issues like the inability to determine if a
+        // logged-in admin (acting on behalf of a member) has already liked a comment.
+        // To resolve this, we retrieve the `impersonate_member_uuid` from the request params and
+        // explicitly set it in the context options as the acting member's ID.
+        // Note: This approach is applied to several admin routes where member context is required.
```

Comment:
It would be good to explain _why_ member context is needed with the comments-ui "use Admin API to replace Members API" use-case, especially as that's why there's a CORS problem.